ddl/external_search_imagerec.hql

```diff
@@ -0,0 +1,26 @@
+-- DDL to create an external table that exposes the
```

Comment:
Nicely documented!

If we migrate to generating and storing parquet data with Spark, I wonder if `saveAsTable()` would let us get away without having to manually create the DDL for this (and other) Hive table.

Reply:
Totally forgot about that. I can test it out

Reply:
Terrific if you have time to give it a go, but I would not consider it a blocker for this PR.
It would be good to standardise on spark in the airflow refactoring, but we can prob live with some hive ddl while we are in bash world.